packages/data-explorer-ui/src/components/Layout/components/Header/components/ProfileComponent/components/authenticationMenu.tsx

```diff
@@ -0,0 +1,65 @@
+import { IconButton, Menu, MenuItem } from "@mui/material";
+import React, { MouseEvent, useState } from "react";
+import { UserProfile } from "../../../../../../../providers/authentication";
+import * as Styled from "./authenticationMenu.styles";
+
+export interface AuthenticationMenuProps {
+  menuPosition: "left" | "right";
+  onLogout: () => void;
+  userProfile: UserProfile;
+}
+
+export const AuthenticationMenu = ({
+  menuPosition = "left",
+  onLogout,
+  userProfile,
+}: AuthenticationMenuProps): JSX.Element => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLButtonElement>(null);
+  const open = Boolean(anchorEl);
+  const growTo = menuPosition === "left" ? "right" : "left";
+
+  const onOpenMenu = (event: MouseEvent<HTMLButtonElement>): void => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const onCloseMenu = (): void => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
+      <IconButton onClick={onOpenMenu}>
+        <Styled.Avatar
+          alt={`${userProfile.given_name} ${userProfile.family_name}`}
+          src={userProfile.picture}
+        />
+      </IconButton>
+      <Menu
+        anchorEl={anchorEl}
+        anchorOrigin={{ horizontal: growTo, vertical: "bottom" }}
+        autoFocus={false}
+        onClose={onCloseMenu}
+        open={open}
+        transformOrigin={{
```

Comment:
If we add `PaperProps={{ variant: "menu" }}` to <Menu> we should be able to get the border, border-radius, and box-shadow - similar to [NavLinkMenu](https://github.com/clevercanary/data-explorer/blob/main/packages/data-explorer-ui/src/components/Layout/components/Header/components/NavLinkMenu/navLinkMenu.tsx). See the Paper theme [here](https://github.com/clevercanary/data-explorer/blob/263afbd4982d8bdf920ffc89bf79037383a71828/packages/data-explorer-ui/src/theme/common/components.ts#L864).